docs/specification/objects.md

```diff
@@ -762,3 +762,23 @@ Field name    | Value type | Default | Description
     "performed_date": "2015-03-14T15:09:27"
 }
 ```
+
+
+## Common Motifs
+
+### Multiple Outputs
+
+A process that produces many output Materials should be characterized through
+individual Process Runs for each output Material, even if the process occurred
+in the same piece of equipment.
+
+![Multi-output process](img/multiple-outputs.png)
```

Comment:
This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.